docs/millennium-os/manual/chapters/30_usage.md

```diff
@@ -151,14 +151,14 @@ options:
                         Show gcode in FreeCAD Editor before saving to file.
   --output-job-setup, --no-output-job-setup
                         When enabled, the post-processor will output
-                        supplemental commands to make sure the machine is
+                        supplemental commands to ensure the machine is
```

Comment:
Please don't replace anything in this code block directly, it's copy and pasted from the help text produced by the post-processor itself. It probably needs updating now that later versions of the post-processor are available with some new features.